bno.cpp

```diff
@@ -0,0 +1,36 @@
+#include "bno.h"
+
+Bno::Bno()
+{
+    bnoSensor = Adafruit_BNO055(55);
+}
+
+//Obtain an int with the X value of the BNO055.
+void Bno::checkAngle()
+{
+    sensors_event_t event;
+    bnoSensor.getEvent(&event);
+    bnoSensor.angle = event.orientation.x;
+    lcd.clear();
```

Comment:
where is lcd decleared?
Maybe you should declared it in the setup and send it in the constructor of BNO class.